.gitignore

```diff
@@ -4,3 +4,4 @@ bin/
 .classpath
 .project
 .settings/
+.idea/
```

Comment:
This should be part of your global git-ignore file, not in local projects. At the very least, it should not be part of this PR, so please revert this change.